### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by florianv**
Updating readme

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -2,41 +2,57 @@
...
-You can find out more about Joomla development *(though not framework specific)* at: http://docs.joomla.org/Developers
+The `Joomla Framework` is a platform for writing Web and command line applications in PHP.  It is free and open source software,
+distributed under the GNU General Public License version 2 or later.
 
-You can discuss *Joomla Framework* development at: http://groups.google.com/group/joomla-dev-platform
+The Joomla Content Management System (CMS) is built on top of the `Joomla Framework`.
+For more information about the Joomla CMS visit: http://www.joomla.org/about-joomla.html
 
-A general guide for contributing to the Joomla platform can be found in the CONTRIBUTING.markdown file in this repository. A thorough contributing guide can be found at: https://github.com/joomla/joomla-platform/wiki/Contributing-to-the-joomla-platform
+You can find out more about Joomla development at: http://docs.joomla.org/Developers.
```
Comment (eddieajau) [inline README.markdown:13]: I think it would be better to link to http://developer.joomla.org

File: README.markdown [hunk 2]
```diff
@@ -2,41 +2,57 @@
...
 
-* [Testing](docs/testing.md)
-* [Analysis](docs/analysis.md)
+All kind of contributions are welcome,
+please consult the document about how to contribute [here](CONTRIBUTING.markdown).
 
-## Coding Standards
+## Running Unit Tests
 
-* [Preface](docs/coding-standards/preface.md)
```
Comment (eddieajau) [inline README.markdown:38]: Why did all the cross linking get removed?

Comment (florianv) [inline README.markdown:38]: They got replaced by the link pointing to the docs folder. I think it's clearer without too much informations

Comment (eddieajau) [inline README.markdown:38]: Ok, I see what you've done.

Comment (florianv) [inline README.markdown:38]: Going to update the developer link, I can add the links back if you want.

Comment (eddieajau) [inline README.markdown:38]: No, it's ok for now. Some of those links might be better in the CONTRIBUTING file anyway. People just wanting to "use" the FW don't need all that distracting information. It's all good.

Comment (florianv) [inline README.markdown:38]: Yes I think people interested in seeing the history might already have looked at the docs.

**Pull request merged**